### PR TITLE
Updating dependencies 

### PR DESCRIPTION
**Pull request by erikd-ambiata**
! @nhibberd @olorin 
/jury approved @nhibberd

### CODE REVIEW

**Comment by olorin** [comment]
Need to update the lockfile for new amazonka version?

**Comment by erikd-ambiata** [comment]
Not just that unfortunately.

File: ambiata-spine.cabal [hunk 1]
```diff
@@ -16,14 +16,14 @@ library
                      , ambiata-p
                      , ambiata-mismi-core
                      , ambiata-x-eithert
-                     , amazonka-dynamodb               >= 1.3.6      && < 1.5
+                     , amazonka-dynamodb               == 1.5.*
```
Comment (nhibberd) [inline ambiata-spine.cabal]: Can we keep the bounds, even with the submodule?

**Comment by nhibberd** [comment]
💯

**Pull request merged**